plugins/lando-services/services/mongo/builder.js

```diff
@@ -8,8 +8,9 @@ module.exports = {
   name: 'mongo',
   config: {
     version: '4.0',
```

Comment:
@rikvanderkemp based on your comments on #2246 re: status of 4.0 and 4.1 does it make sense to set 4.2 as the default version?

Reply:
Great point! 

Especially when I look at the following page:
 https://www.mongodb.com/support-policy

4.0 and 4.2 are both TBD while 4.1 isn't mentioned.

4.0 is from June 2018 and 4.2 August 2019.

Based on this I would say set 4.2 to default.

Maybe even dropping 4.1 altogether and add a deprecation warning for it. @pirog did something similar to phpmyadmin.

What do you think?

Reply:
@rikvanderkemp was looking at exactly the same. yes i think moving 4.1 into the "you can use it still but it throws a warning" group makes sense.

im merging a bunch of stuff together rn so im happy to make these changes

Reply:
Perfect that would be great.